pkg/project/project.go

```diff
@@ -1,7 +1,7 @@
 package project
 
 var (
-	bundleVersion        = "7.0.1-dev"
+	bundleVersion        = "7.0.1-rc1"
```

Comment:
Is this the version we want or should it be maybe `7.1.0-rc1` or `8.0.0-rc1` (it's not backwards compatible with earlier node pool version)?

Reply:
`8.0.0-rc1` makes sense to me.